### PR TITLE
Adds custom amq_streams_zookeeper_zookeeper_id to zookeeper.properties.j2

### LINKED ISSUE #100: Zookeeper ids are not asigned in the zookeeper ensemble list - Zookeeper does not start
##### SUMMARY

The new feature of identify the zookeeper id works if you use a sequence of ids starting from 0, but if you use a difference sequence, then zookeeper does not start and fails with the following exception:

```
[2023-10-12 17:51:33,862] ERROR Unexpected exception, exiting abnormally (org.apache.zookeeper.server.quorum.QuorumPeerMain)
java.lang.RuntimeException: My id 30 not in the peer list
	at org.apache.zookeeper.server.quorum.QuorumPeer.start(QuorumPeer.java:1077)
	at org.apache.zookeeper.server.quorum.QuorumPeerMain.runFromConfig(QuorumPeerMain.java:227)
	at org.apache.zookeeper.server.quorum.QuorumPeerMain.initializeAndRun(QuorumPeerMain.java:136)
	at org.apache.zookeeper.server.quorum.QuorumPeerMain.main(QuorumPeerMain.java:90)
[2023-10-12 17:51:33,868] INFO ZooKeeper audit is disabled. (org.apache.zookeeper.audit.ZKAuditProvider)
```

This is because the list of servers of the zookeeper cluster has not the same id (based in the zookeeper group index). This is the list created, not linked with the id declared in the `myid` file: 

```properties
# List of servers which should be members of the Zookeeper cluster.
server.1=f38mw01:2888:3888:participant;f38mw01:2181
server.2=f38mw02:2888:3888:participant;f38mw02:2181
server.3=f38mw03:2888:3888:participant;f38mw03:2181
```

This also affects authorization provider if authentication is enabled:

```properties 
# Client-to-Server Authentication
requireClientAuthScheme=sasl
authProvider.1=org.apache.zookeeper.server.auth.SASLAuthenticationProvider
authProvider.2=org.apache.zookeeper.server.auth.SASLAuthenticationProvider
authProvider.3=org.apache.zookeeper.server.auth.SASLAuthenticationProvider
```

##### ISSUE TYPE
 - Bug Report

##### ANSIBLE VERSION

```
ansible [core 2.14.10]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/rmarting/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.11/site-packages/ansible
  ansible collection location = /home/rmarting/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.11.5 (main, Aug 28 2023, 00:00:00) [GCC 13.2.1 20230728 (Red Hat 13.2.1-1)] (/usr/bin/python3)
  jinja version = 3.0.3
  libyaml = True
```

##### COLLECTION VERSION

Current content of the `main` branch

##### STEPS TO REPRODUCE

The following inventory will fails with this issue as the ids are not declared sequentialy 

```yaml
zookeepers:
  hosts:
    f38mw01:
      amq_streams_zookeeper_zookeeper_id: 10
    f38mw02:
      amq_streams_zookeeper_zookeeper_id: 20
    f38mw03:
      amq_streams_zookeeper_zookeeper_id: 30
```

This can happen with any playbook installing multiple zookeepers.

##### EXPECTED RESULTS

Use the zookeeper id per host in the list of servers of the ensemble (including the autorization providers).

##### ACTUAL RESULTS

```
TASK [amq_streams_common : Check if service is started] ********************************************************************************************************************************************************************************************************
fatal: [f38mw01]: FAILED! => {"assertion": "ansible_facts.services[\"amq_streams_zookeeper.service\"]['state'] == 'running'", "changed": false, "evaluated_to": false, "msg": "Service is not started."}
fatal: [f38mw02]: FAILED! => {"assertion": "ansible_facts.services[\"amq_streams_zookeeper.service\"]['state'] == 'running'", "changed": false, "evaluated_to": false, "msg": "Service is not started."}
fatal: [f38mw03]: FAILED! => {"assertion": "ansible_facts.services[\"amq_streams_zookeeper.service\"]['state'] == 'running'", "changed": false, "evaluated_to": false, "msg": "Service is not started."}
```

### PR DESCRIPTION
**Pull request by gbaufake**
- Fixes #100

### CODE REVIEW

**Comment by rmarting** [comment]
@gbaufake Did you test it? The implementation looks like good to me, but I would like to confirm if you tested the implementation with a playbook in a distributed environment. For example, using the configuration described in the issue.

Thanks

**Comment by gbaufake** [comment]
@rmarting Give me some hours and I can put here the test results.

**Comment by rpelisse** [comment]
@gbaufake Did the fix worked?

**Comment by gbaufake** [comment]
Found some corner cases, I will work on that

**Comment by gbaufake** [comment]
Great news! I have successfully resolved the issue related to checking the default variable `amq_streams_zookeeper_zookeeper_id`. 

I hope it is correct now.

File: galaxy.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ---
 namespace: middleware_automation
 name: amq_streams
-version: "0.0.7"
+version: "0.0.8"
```
Comment (rmarting) [inline galaxy.yml:4]: @rpelisse The galaxy version is not updated manually, right? I mean, it is implemented by the release pipeline with a new tag is created, so we don't need to update the `galaxy.yml` file? Just to double check before merging this PR.

**Review by rmarting — CHANGES_REQUESTED** [review]

File: galaxy.yml (hunk 1 above)
Comment (gbaufake) [reply galaxy.yml:4]: @rmarting if you want, I can rollback that change

**Comment by rpelisse** [comment]
@gbaufake @rmarting Where are we on this PR?

**Comment by gbaufake** [comment]
From my part, we are done, and we could merge it.

I tested it when the zookeeper id was filled and the zookeeper id was not filled.

**Comment by rmarting** [comment]
LGTM!
